core/src/test/java/com/linecorp/armeria/client/endpoint/dns/TestDnsServer.java

```diff
@@ -63,6 +64,11 @@ public static DnsRecord newAddressRecord(String name, String ipAddr, long ttl) {
     private volatile Map<DnsQuestion, DnsResponse> responses;
 
     public TestDnsServer(Map<DnsQuestion, DnsResponse> responses) {
+        this(responses, new ChannelInboundHandlerAdapter());
```

Comment:
How about passing `null` and not adding anything?

Reply:
Fixed. 😄